contracts/marketplace/Makefile.toml

```diff
@@ -0,0 +1,231 @@
+[env]
+friendbot = "https://friendbot-futurenet.stellar.org/"
+network = "futurenet"
```

Comment:
Should we start using `testnet` instead of futurenet ? 

Looks like futurenet could be disabled at some point, now that we are in testnet.